pvsystemprofiler/scripts/latitude_script.py

```diff
@@ -59,7 +64,7 @@ def evaluate_systems(df, df_ground_data, power_column_label, site_id, time_shift
                     if manual_time_shift == 1:
                         dh.fix_dst()
 
-                passes_pipeline = run_failsafe_pipeline(dh, df, sys_tag, fix_time_shifts, time_zone_correction)
+                passes_pipeline = run_failsafe_pipeline(df, dh, sys_tag, fix_time_shifts, time_zone_correction)
```

Comment:
move data handler instantiation into `run_failsafe_pipeline`. check downstream effects to other scripts